test/validation/api/timer/timer.c

```diff
@@ -334,8 +334,8 @@ static void timer_test_queue_type(odp_queue_type_t queue_type)
 			tim = odp_timeout_timer(tmo);
 			tick = odp_timeout_tick(tmo);
 
-			CU_ASSERT(diff_period > (period_ns - (4 * res_ns)));
-			CU_ASSERT(diff_period < (period_ns + (4 * res_ns)));
+			CU_ASSERT(diff_period > (period_ns - (5 * res_ns)));
+			CU_ASSERT(diff_period < (period_ns + (5 * res_ns)));
```

Comment:
Not sure what the best way to deal with this is, but these sort of ad-hoc "fudge factors" seem awkward.

Reply:
Yes. Moreover they still do not guarantee stable behaviour. I'm playing with the code to fix such kinds of tests.